.circleci/config.yml

```diff
@@ -10,53 +10,19 @@ aliases:
         ignore: /^.*-SNAPSHOT/
       branches:
         ignore: /.*/
+  release-branches: &release-branches
+    filters:
+      tags:
+        ignore: /.*/
+      branches:
+        only: /^release\/.*/
```

Comment:
I wonder, can these aliases be moved to the orbs as well? Since they are pretty much the same in all repos

Reply:
I would need to check it out but I don't think aliases is something we can add to the orb. I know it wasn't supported when I was working on the monorepo and spliting the yaml in different yamls